src/frontend/utils/validators.ts

```diff
@@ -50,12 +50,11 @@ export const erGyldigMånedDato = (
 
 const barnsVilkårErMellom0og18År = (fom: string, person: IGrunnlagPerson, tom?: string) => {
     const fødselsdato = familieDayjs(new Date(person.fødselsdato));
-    const fødselsdatoPluss18 = leggTilÅr(person.fødselsdato, 18);
+    const fødselsdatoPluss18 = leggTilÅr(person.fødselsdato, 18).subtract(1, 'day');
     const fomDato = familieDayjs(new Date(fom));
     const tomDato = tom ? familieDayjs(new Date(tom)) : undefined;
     return (
-        fomDato.isSameOrAfter(fødselsdato) &&
-        (tomDato ? tomDato.isSameOrBefore(fødselsdatoPluss18) : true)
+        fomDato.isSameOrAfter(fødselsdato) && (tomDato ? tomDato.isSame(fødselsdatoPluss18) : true)
```

Comment:
```suggestion
        fomDato.isSameOrAfter(fødselsdato) && (tomDato ? tomDato.isSameOrBefore(fødselsdatoPluss18) : true)
```

hvis den brukes på andre vilkår må man vel fortsatt få lov til å ha perioder kortere enn til 18 år

Reply:
@ninakjek Skjønner godt du spør :-) Forslaget ditt er sånn vi hadde det, men ved utregning av perioder for tilkjent ytelse gjør vi for 18-årsvilkåret nå følgende: Vi legger på en dag og trekker fra en måned. Dette blir da riktig _kun_ dersom tom er satt til dagen før barnet fyller 18 år, og derfor må vi ha denne valideringen på det. Iflg fag gir det heller ikke mening å forkorte ned 18-årsvilkåret.